plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -372,6 +371,7 @@ func (b *backend) pathVenafiCertObtain(ctx context.Context, req *logical.Request
 	expirationSec := expirationTime.Unix()
 
 	respData := map[string]interface{}{
+		"certificate_uid":   certId,
```

Comment:
Please put here a comment explaining the different kind of source of this certificate_uid

Reply:
Got it

Reply:
This is still confusing to the user, users are not looking at code comments. 
I would prefer to see a `source` or `origin` key that has the value of the `store_by` property. That way the user has easy access to understand where `certificate_uid` comes from. And would drop `certificate_uid` for `certificate_id` unless there is a conflict in namings